tests/integration/test_webp_optimization_integration.py

```diff
@@ -0,0 +1,47 @@
+"""Integration tests for WebP optimization insight."""
+
+from pathlib import Path
+from typing import cast
+
+import pytest
+
+from launchpad.analyzers.android import AndroidAnalyzer
+from launchpad.artifacts.artifact import AndroidArtifact
+from launchpad.artifacts.artifact_factory import ArtifactFactory
+from launchpad.models.android import OptimizeableImageFile, WebPOptimizationInsightResult
+
+
+class TestWebPOptimizationInsight:
+    """Test WebP optimization insight functionality."""
+
+    @pytest.fixture
+    def sample_android_apk_path(self) -> Path:
+        """Path to sample Android APK for testing."""
+        return Path("tests/_fixtures/android/app-debug.apk")
```

Comment:
I don't see where app-release.apk is used - did you mean to add that? Also you could leverage hn.apk potentially, up to you. Not sure what's in these APKs